runtime/mandala/src/benchmarking/cdp_engine.rs

```diff
@@ -197,14 +195,8 @@ runtime_benchmarks! {
 		let collateral_amount = Price::saturating_from_rational(dollar(LIQUID), dollar(STABLECOIN)).saturating_mul_int(collateral_value);
 		let collateral_price = Price::one();		// 1 USD
 
-		path.insert(0, LIQUID);
-		for i in 0..path.len() {
-			if i != 0 {
-				inject_liquidity(funder.clone(), path[i], path[i-1], 10_000 * dollar(path[i]), 10_000 * dollar(path[i-1]))?;
-			}
-		}
-
 		set_balance(LIQUID, &owner, (10 * collateral_amount) + ExistentialDeposits::get(&LIQUID));
+		inject_liquidity(funder, LIQUID, STABLECOIN, 10_000 * dollar(LIQUID), 10_000 * dollar(STABLECOIN))?;
 
```

Comment:
same here acala/karura worst case LIQUID => STAKING => STABLECOIN. We could always do a followup pr as it doesn't effect any  logic